octobot_tentacles_manager/creators/tentacle_creator.py

```diff
@@ -13,27 +13,26 @@
 #
 #  You should have received a copy of the GNU Lesser General Public
 #  License along with this library.
-
-import logging
 import os
-
+from logging import INFO
 from jinja2.nativetypes import NativeEnvironment
 
-from octobot_tentacles_manager import TENTACLE_TEMPLATE_PATH, \
-    TENTACLE_TEMPLATE_DESCRIPTION, TENTACLE_TEMPLATE_EXT, TENTACLE_TEMPLATE_PRE_EXT, TENTACLE_PARENTS, TENTACLE_SONS, \
-    EVALUATOR_ADVANCED_FOLDER, TENTACLES_PATH, TENTACLE_CONFIG_TEMPLATE_PRE_EXT, CONFIG_FILE_EXT, \
-    EVALUATOR_CONFIG_FOLDER, INFO
-from octobot_tentacles_manager.util.tentacle_util import get_tentacles_arch
+from octobot_commons.logging.logging_util import get_logger, set_global_logger_level
+from octobot_tentacles_manager.constants import TENTACLE_TEMPLATE_PATH, \
+    TENTACLE_TEMPLATE_DESCRIPTION, TENTACLE_TEMPLATE_EXT, TENTACLE_TEMPLATE_PRE_EXT, \
+    TENTACLES_PATH, TENTACLE_CONFIG_TEMPLATE_PRE_EXT, CONFIG_EXT, \
+    TENTACLE_CONFIG, TENTACLES_FOLDERS_ARCH
+
+# TODO: remove from to .coveragerc when adapted
 
 
 class TentacleCreator:
     def __init__(self, config):
-        self.tentacles_arch, _ = get_tentacles_arch()
         self.config = config
         self.templates = {}
         self.config_templates = {}
-        self.logger = logging.getLogger(self.__class__.__name__)
-        self.logger.setLevel(INFO)
+        self.logger = get_logger(self.__class__.__name__)
+        set_global_logger_level(INFO)
```

Comment:
Why do you need this ?

Reply:
It was like this in the previous version, I just made it in line with what we usualy do. Might be useless, TBD when refactoring the creator